docs/user/backends.md

```diff
@@ -73,6 +78,33 @@ healthy.
 SERVICE_CHECK_TTL=30s
 ```
 
+### Consul Docker Check
+
+This feature lets you register Docker checks with Consul, so your checks can
+run within existing containers instead of from the Consul agent.
+
+```bash
+SERVICE_DOCKER_CHECK_SCRIPT=/path/to/container/check
```

Comment:
@threetee any chance you can address this? Would really like to use this functionality :)